links_test.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "category": "Marilyn Manson",
+    "title": "Rock Is Dead",
+    "date": "1998.xx.xx",
+    "url": "https://youtu.be/vEc9nXErU-Y",
+    "comment": "- Rock! is deader than dead!"
+  },
+  {
+    "category": "Bon Jovi",
+    "title": "Always",
+    "date": "1994.xx.xx",
+    "url": "https://youtu.be/9BMwcO6_hyA",
+    "comment": "- I've made mistakes, I'm just a man"
+  },
+  {
+    "category": "Mr. Big",
+    "title": "To Be With You",
+    "date": "1991.xx.xx",
+    "url": "https://youtu.be/L6-uJLteKek",
+    "comment": "- Build up your confidence<br>- So you can be on top for once"
+  },
+  {
+    "category": "Skid Row",
+    "title": "I Remember You",
+    "date": "1989.xx.xx",
+    "url": "https://youtu.be/qjuEXKwnkLE",
+    "comment": "- Live for your smile and die for your kiss!"
+  },
+  {
+    "category": "……",
+    "title": "……",
+    "date": "……",
+    "url": "",
+    "comment": "- ……"
```

Comment:
**Rejected** : It requires modifying the border radius of `row1`, but it would be too much work. And, actually, the possibility of a case without a comment is quite rare.